carbon-design-system/carbon-addons-iot-react
# fix(date-time-picker):  fix relative to arrow arrow bug

Closes #2616 

**Summary**

- The arrow on "Relative to" was too far to the right. Fix that and the spacing between the arrow and the edge

**Change List (commits, features, bugs, etc)**

- add helper classes to component
- css fixes for arrow

**Acceptance Test (how to verify the PR)**

- Open the default time picker story
- choose "Custom range"
- Turn off hasTimeInput knob
- dropdown arrow on right of "Relative to" should be within the field

**Regression Test (how to make sure this PR doesn't break old functionality)**

- nothing else changes.



**Things to look for during review**
